install/kubernetes/cilium/values.yaml

```diff
@@ -352,6 +352,10 @@ daemon:
 
 enableCnpStatusUpdates: false
 
+# Configures the use of the KVStore to optimize Kubernetes event handling by 
```

Comment:
```suggestion
# -- Configures the use of the KVStore to optimize Kubernetes event handling by 
```

This will also update the description in the README.

Reply:
@fristonio cheers, all updated.

Reply:
Hey @youssefazrak I think you missed to regenerate the helm readme. :sweat_smile: 
Can you run `make` from `install/kuberentes` and push the changes?

https://github.com/cilium/cilium/pull/14555/checks?check_run_id=1665630115

Reply:
@fristonio ha, quite correct! Should be fixed now :)